docs/concepts/CLQL.md

```diff
@@ -1,6 +1,6 @@
 # Lexicons
 
-CodeLingo Query Language (CLQL) queries are statements of Facts about a domain of knowledge. Those Facts come from Lexicons.
+CodeLingo Query Language (CLQL) queries are statements of Facts about a domain of knowledge such as an AST. Those Facts come from Lexicons, for example, an AST lexicon provides Facts about the syntatic structure of a codebase which we can then query to identify complex patterns.
```

Comment:
Hmm, I feel like this could be worded better but I'm not sure how... Might get a second review on this.
```suggestion
CodeLingo Query Language (CLQL) queries are statements of Facts about a domain of knowledge (e.g. an AST). Facts come from Lexicons. In the case of an AST Lexicon, the Lexicon provides Facts about the syntactic structure of a codebase, which can then be queried to identify arbitrary patterns in code.
```